voicevox_engine/tts_pipeline/tts_engine.py

```diff
@@ -220,6 +237,12 @@ def query_to_decoder_feature(
 
     # 設定を適用する
     moras = apply_prepost_silence(moras, query)
+    if (
+        query.pauseLength is not None and query.pauseLength >= 0
+    ):  # pauseLengthがNoneでない場合
+        moras = apply_pause_length(moras, query)
+    if query.pauseLengthScale != 1:  # pauseLengthScaleが1以外の場合
+        moras = apply_pause_length_scale(moras, query)
```

Comment:
コメントが間違ってそうなのと、他に合わせてまあifは削れるかも。
（あっても良いのですが、周りに合わせるという意味で）
```suggestion
    if (
        query.pauseLength is not None and query.pauseLength >= 0
    ):
        moras = apply_pause_length(moras, query)
    moras = apply_pause_length_scale(moras, query)
```